ping.go

```diff
@@ -101,6 +107,26 @@ func NewPinger(addr string) (*Pinger, error) {
 	}, nil
 }
 
+// PingerConf represents options to configuring a pinger.
+type PingerConf struct {
+	// The address to ping.
+	Address string
+	// The network type from net.ResolveIPAddr(). ("ip", "ip4", "ip6")
+	Network string
+}
+
+func NewPingerConf(n string, addr string) PingerConf {
```

Comment:
I would prefer if the arguments to this function only included required parameters.